### DISCUSSION TITLE
Rootless dind latest

### DISCUSSION THREAD

**Discussion by imbabamba** [some-natalie/kubernoodles#278] [General]
First of all, you're awesome! Such an inspiration for the github runner and all community .

I use your numbat rootless image. Was wondering why you don't use semver for your releases additionally to the latest tag

**Comment by some-natalie** [comment]
Thanks @imbabamba !

I typically love semver, but never felt it was the right choice for CI images.  These images don't typically change because of changes in the software included in the image, but are rebuilt weekly instead.  `latest` gives you whatever the latest is, which is typically desired here.  However each build also is tagged with the git short SHA (eg `1cbbd14`) to give you a human-friendly stable tag.  SHA digest is there too. :)

**Reply by imbabamba** [reply]
@some-natalie thanks for answering :)

Hmm makes sense ofc. but reasons for questions are that we use dependabot , when pointing to image we typically would want a PR suggesting there's an update but if doesn't work that good since the versions aren't semver. But maybe im doing it all wrong :D

**Reply by some-natalie** [reply]
For sure - Dependabot doesn't pick that up.  FWIW, the company I work for wrote a small open source utility to solve exactly this problem called [Digestabot](https://github.com/chainguard-dev/digestabot).  It uses tag+SHA to open a pull request whenever the SHA changes on a given tag.

**Reply by imbabamba** [reply]
> For sure - Dependabot doesn't pick that up. FWIW, the company I work for wrote a small open source utility to solve exactly this problem called [Digestabot](https://github.com/chainguard-dev/digestabot). It uses tag+SHA to open a pull request whenever the SHA changes on a given tag.

hey Nat. sadly it didn't exactly do what we wanted but thanks a lot for reaching out and submitting the tool . For now dependabot will look in small dockerfiles only containing the 'FROM' like `FROM ghcr.io/actions/actions-runner:2.320.0`
